internal/mode/static/state/conditions/conditions.go

```diff
@@ -18,7 +18,7 @@ const (
 	// ListenerMessageFailedNginxReload is a message used with ListenerConditionProgrammed (false)
 	// when nginx fails to reload.
 	ListenerMessageFailedNginxReload = "The Listener is not programmed due to a failure to " +
-		"reload nginx with the configuration"
+		"reload nginx with the configuration. Please see the nginx container logs for any possible configuration issues."
```

Comment:
Part of the AC for #664 is that if a reload fails, we should report the error to the user.

Is that possible as part of this work? If not, should we open a separate ticket for that?

Reply:
Part of the AC is to return _an_ error to the user for a failed reload, not _the_ error (pedantry I know 🤣) but the distinction is important. Getting the error message from NGINX could only (I think) be achieved from scraping/ teeing logs in some way. We can theoretically access pod logs, but it feels a bit brittle to me. Is it critical that we return the error from NGINX in the NKG logs and/ or in the conditions?

Reply:
Errors are intended to aid the user in overcoming the condition.
I think the intention should be to provide the error from NGINX as a part of the message to the customer.
However, when (now vs later) and how (maybe another future change will make this easier) this is delivered is the thing that needs to be considered.

Reply:
As discussed offline - Returning the error itself as part of the status is not critical at this stage. We should create a follow on ticket for this - there is definitely value in returning the reload error in the status, but how we achieve this is still a question.